charts/steward/templates/deployment-run-controller.yaml

```diff
@@ -33,9 +33,13 @@ spec:
         image: {{ printf "%s:%s" .repository .tag | quote }}
         imagePullPolicy: {{ .pullPolicy | quote }}
         {{- end }}
-        args:
-        - -qps={{ .Values.runController.args.qps }}
-        - -burst={{ .Values.runController.args.burst }}
+        args: [
+        "-qps={{ .Values.runController.args.qps }}",
+        "-burst={{ .Values.runController.args.burst }}",
+        {{- if .Values.runController.args.logVerbosity }}
+        "-v", {{ .Values.runController.args.logVerbosity | quote }},
+        {{- end }}
+        ]
```

Comment:
The JSON syntax is invalid: a trailing comma in a list is not allowed by the official [JSON grammar](https://www.json.org/json-en.html).

Reply:
copy paste error from other project.
I approved your https://github.com/SAP/stewardci-core/pull/144

Reply:
Not really an error since it is yaml (superset of json). The generated result is correct without trailing comma.